src/Features/Core/Portable/FileHeaders/FileHeader.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.FileHeaders
+{
+    /// <summary>
+    /// Contains the parsed file header information for a syntax tree.
+    /// </summary>
+    internal readonly struct FileHeader
+    {
+        private readonly int _fileHeaderStart;
+        private readonly int _commentPrefixLength;
```

Comment:
consider doc'ing this field.   it's not entirely clear ot me how to interpret it. i.e. i could imagine it's the spaces in `//   foo`  or hte spaces in `    //xyz`

Reply:
It's actually the length of `//`. For C# it's 2 (`//`), and for VB it's 1 (`'`).

Reply:
got it.  def doc.  that wasn't clear :D